src/Logging/Logging.Abstractions/src/EventId.cs

```diff
@@ -3,42 +3,80 @@
 
 namespace Microsoft.Extensions.Logging
 {
+    /// <summary>
+    /// Identifies a logging event.
```

Comment:
I think it would be very useful to add some explanatory content here because I can never remember what matters here: does the int matter, does the string matter, do both matter?

By reading the code, it looks like the int is required and is what should be unique, and that the string is optional and doesn't need to be unique? Is that even true? Does anyone know? 😄 We should figure it out and document it here.